modules/my_health/spec/serializer/v1/prescription_details_serializer_spec.rb

```diff
@@ -0,0 +1,172 @@
+  end
+
+  it 'includes the cmop_ndc_number' do
+    expect(attributes['cmop_ndc_number']).to eq(prescription.cmop_ndc_number)
+  end
+
+  it 'includes the sig' do
+    expect(attributes['sig']).to eq(prescription.sig)
+  end
+
+  it 'includes the user_id' do
+    expect(attributes['user_id']).to eq(prescription.user_id)
+  end
+
+  it 'includes the provider_first_name' do
+    expect(attributes['provider_first_name']).to eq(prescription.provider_first_name)
+  end
+
+  it 'includes the provider_last_name' do
+    expect(attributes['provider_last_name']).to eq(prescription.provider_last_name)
+  end
+
+  it 'includes the remarks' do
+    expect(attributes['remarks']).to eq(prescription.remarks)
+  end
+
+  it 'includes the division_name' do
+    expect(attributes['division_name']).to eq(prescription.division_name)
+  end
+
+  it 'includes the modified_date' do
+    expect(attributes['modified_date']).to eq(prescription.modified_date.strftime('%Y-%m-%dT%H:%M:%S.%LZ'))
+  end
+
+  it 'includes the institution_id' do
+    expect(attributes['institution_id']).to eq(prescription.institution_id)
+  end
+
+  it 'includes the dial_cmop_division_phone' do
+    expect(attributes['dial_cmop_division_phone']).to eq(prescription.dial_cmop_division_phone)
+  end
+
+  it 'includes the disp_status' do
+    expect(attributes['disp_status']).to eq(prescription.disp_status)
+  end
+
+  it 'includes the ndc' do
+    expect(attributes['ndc']).to eq(prescription.ndc)
+  end
+
+  it 'includes the reason' do
+    expect(attributes['reason']).to eq(prescription.reason)
+  end
+
+  it 'includes the prescription_number_index' do
+    expect(attributes['prescription_number_index']).to eq(prescription.prescription_number_index)
+  end
+
+  it 'includes the prescription_source' do
+    expect(attributes['prescription_source']).to eq(prescription.prescription_source)
+  end
+
+  it 'includes the disclaimer' do
+    expect(attributes['disclaimer']).to eq(prescription.disclaimer)
+  end
+
+  it 'includes the indication_for_use' do
+    expect(attributes['indication_for_use']).to eq(prescription.indication_for_use)
+  end
+
+  it 'includes the indication_for_use_flag' do
+    expect(attributes['indication_for_use_flag']).to eq(prescription.indication_for_use_flag)
+  end
+
+  it 'includes the category' do
+    expect(attributes['category']).to eq(prescription.category)
+  end
+
+  it 'includes the tracking' do
+    expect(attributes['tracking']).to eq(prescription.tracking)
+  end
+
+  it 'includes the orderable_item' do
+    expect(attributes['orderable_item']).to eq(prescription.orderable_item)
+  end
+
+  it 'includes the sorted_dispensed_date' do
+    expect(attributes['sorted_dispensed_date']).to eq(prescription.sorted_dispensed_date.strftime('%Y-%m-%d'))
+  end
+
+  it 'includes the rx_rf_records' do
+    expect(attributes['rx_rf_records']).to be_an(Array)
+    expect(attributes['rx_rf_records']).to all(be_a(Hash)) if attributes['rx_rf_records'].present?
```

Comment:
I wonder if it would be ideal to ensure the test factory has rx_rf_records available (as it does currently), or building a record that does specifically for this test?

I think the `if attributes['rx_rf_records'].present?` shouldn't be needed... it is signaling uncertainty about test data that we should be able to control

Reply:
Agreed, removing `.present?`